resources/js/components/DeviceRepairStatus.vue

```diff
@@ -104,6 +104,12 @@ export default {
     },
   },
   computed: {
+    translatedBarriers() {
+      return this.barrierList.map(b => {
+        b.barrier = this.$lang.get('_json.' + b.barrier)
```

Comment:
I think this should be `.get('strings.' + b.barrier)`

Reply:
You're right, and that works.  I'm not sure how it was working for fr-BE before.